src/Runner.ts

```diff
@@ -1,24 +1,50 @@
-import { ProjectFileType } from '@dojo/cli-export-project/interfaces/project.json';
-import Evented from '@dojo/core/Evented';
 import { createHandle } from '@dojo/core/lang';
-import project from './project';
+import { v } from '@dojo/widget-core/d';
+import { DNode, WidgetProperties } from '@dojo/widget-core/interfaces';
+import WidgetBase, { afterRender } from '@dojo/widget-core/WidgetBase';
+import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mixins/Themeable';
+import { Program } from './project';
+import * as css from './styles/runner.m.css';
 import * as base64 from './support/base64';
 import DOMParser from './support/DOMParser';
 import { wrapCode } from './support/sourceMap';
 
-export interface GetDocOptions {
-	css?: { name: string; text: string; }[];
-	bodyAttributes?: { [attr: string]: string; };
-	dependencies: { [pkg: string]: string; };
-	loaderSrc: string;
-	html?: string;
-	modules: { [mid: string]: { code: string; map: string; } };
-	scripts?: string[];
+/**
+ * @type RunnerProperties
+ *
+ * Properties that can be set on a Runner widget
+ *
+ * @property loader A URI that points to an AMD loader which will be used when running the program.
+ *                  Defaults to `https://unpkg.com/@dojo/loader/loader.min.js`
+ * @property src A URI that points to the `src` to set on the Runner's `iframe`. Defaults to
+ *               `../support/blank.html`
+ * @property onError A method that will be called whenever there is an error in the running program
+ * @property onInitIframe A method that will be called when the `iframe` is initiailized
+ * @property onRun A method that will be called when the `Runner` has fully loaded the program.  *Note* this does not
+ *                 represent the state of the running program, it simply indicates that the `Runner` no longer has
+ *                 involvement in the process of loading the program
+ */
+export interface RunnerProperties extends Partial<Program>, WidgetProperties, ThemeableProperties {
+	loader?: string;
+	src?: string;
+
```

Comment:
Okay same question re line break but I think I see the pattern now separating values and functions.